src/Redux/books.js

```diff
@@ -0,0 +1,27 @@
+const ADDBOOK = 'bookstore/books/ADDBOOK';
+const REMOVEBOOK = 'bookstore/books/REMOVEBOOK';
+
+const booksReducer = (state = {}, action) => {
```

Comment:
- [ ] Good job so far 👍 Kindly set the initial state in both the books and categories redux files to be an empty array 👍

Reply:
The add and remove book changes requested by the reviewer do not fall under this project. this project is a setup to add reducers and actions.